docker-swarm-cluster/azuredeploy.json

```diff
@@ -0,0 +1,256 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "West Europe",
```

Comment:
As a best practice, please remove the default value for location and let the user pick one of the allowed values for scenarios where the parameter file is not specified and clients (PS, CLI) prompt the user for entering a value for paramateres where defaults are not specified